src/Authenticators/LDAPLoginForm.php

```diff
@@ -110,6 +111,17 @@ public function __construct($controller, $name, $fields = null, $actions = null,
      */
     protected function buildRequestHandler()
     {
-        return LDAPMemberLoginHandler::create($this);
+        return LDAPMemberLoginHandler::create($this, Injector::inst()->get(LDAPAuthenticator::class));
+    }
+
+    /**
+     * The name of this login form, to display in the frontend
+     * Replaces Authenticator::get_name()
+     *
+     * @return string
+     */
+    public function getAuthenticatorName()
+    {
+        return _t(self::class . '.AUTHENTICATORNAME', 'LDAP');
```

Comment:
`::class` won't work at the moment with the text collector, you should use `__CLASS__` instead